charts/estafette-k8s-hpa-scaler/templates/NOTES.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
What's the use of this empty file? Is it required or can we leave it out?

Reply:
Apparently is optional. It should have some short usage notes. I'll either add those notes or remove it :+1:

Reply:
I couldn't find anything useful to add to this file, so I'm removing it.